test/router.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 
+const net = require('net')
 const test = require('tap').test
+// const only = require('tap').only
```

Comment:
why is this commented out?

Reply:
Forgot to remove it after I was done with it.